qiskit_superstaq/superstaq_provider_test.py

```diff
@@ -79,6 +80,16 @@ def test_aqt_compile(mock_post: MagicMock) -> None:
     assert not hasattr(out, "circuit") and not hasattr(out, "pulse_list")
 
 
+@mock.patch(
```

Comment:
nit: for consistency with the rest of the file this should just be,
```suggestion
@patch(
```
or (possibly better) we should use `mock.MagicMock` and `mock.patch` everywhere else and get rid of the extra import line